configure.py

```diff
@@ -1053,13 +1021,12 @@ def main():
 
   set_action_env_var(environ_cp, 'TF_NEED_OPENCL_SYCL', 'OpenCL SYCL', False)
   if environ_cp.get('TF_NEED_OPENCL_SYCL') == '1':
-    set_host_cxx_compiler(environ_cp)
-    set_host_c_compiler(environ_cp)
```

Comment:
If these are needed for triSYCL, then these could be moved into the `else` below.